### DISCUSSION TITLE
Key mapping IOsender - flexi hal

### DISCUSSION THREAD

**Discussion by Ed-27** [terjeio/ioSender#429] [Q&A]
Is there some place I can find the keyboard mapping for controlling the CNC and a method to turn it on in iosender?

**Comment by terjeio** [comment]
See the Wiki [here](https://github.com/terjeio/ioSender/wiki/Usage-tips) and [here](https://github.com/terjeio/ioSender/wiki/Setup-and-configuration). Wiki and Usage tips are also linked from the sender _Help_ menu.

**Reply by Ed-27** [reply]
when I installed version 44 I didn't get a version that had the network code, to avoid that again can you tell me name and location of the 45 code file that includes the networking code.  Thank you.

**Reply by terjeio** [reply]
IIRC it is [this library](https://github.com/sta/websocket-sharp). If you are compiling yourself then you do not need the code unless you are going to modify it. Just compile aganst the dll - which you can copy from the latest release.

**Reply by Ed-27** [reply]
I am sorry but this is all Greek to me.  When I loaded up version 2.0.44 I got a zip file that I unpacked and found the iosender.exe file, which I have been running on.  I don't see anything like that here.    Is there such a thing for this version?

**Reply by terjeio** [reply]
_iosender.exe_ and the related files can be found in the _ioSender 2.0.45_ directory in the 
_ioSender.2.0.45.zip_ or _ioSender.XL.2.0.45.zip_ available [here](https://github.com/terjeio/ioSender/releases/tag/2.0.45).
